### PR TITLE
Fixed getuser permission check and added startup script

### PR DESCRIPTION
**Pull request by WeilonYing**
Fixed previously not-working permission check for !getuser command.
Added startup script and deployment instructions

Test plan:
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,27 @@
 # UNSW-Minecraft-Society-Discord-Verification-Bot
 Email verification bot for UNSW Minecraft Society's Discord server
 
+## Development setup
+
```
Comment (inuradz) [inline README.md]: This is a lot nicer than what I wrote

File: utils.js [hunk 2]
```diff
@@ -78,8 +78,9 @@ function is_exec_or_higher(guild_roles, member_roles) {
     if (!exec_role) {
         throw new Error('Exec role not in given guild roles');
     }
-    for (let i = 0; i < member_roles.length; i++) {
-        const current_role = member_roles[i];
+    const key_array = member_roles.keyArray();
+    for (let i = 0; i < key_array.length; i++) {
```
Comment (inuradz) [inline utils.js]: I feel like using a map would make this better than iterating through the values yourself

File: commands/getuser.js [hunk 3]
```diff
@@ -34,7 +34,7 @@ async function execute(state, message, args) {
     // Check command sender is Administrator or has the exec role or higher
     const guildmember = guild.member(message.author);
 
-    if (!utils.is_exec_or_higher(guild.roles, guildmember.roles)
+    if (!utils.is_exec_or_higher(guild.roles, guildmember.roles)) {
```
Comment (inuradz) [inline commands/getuser.js]: This won't work? The and is not going to interact properly

Comment (WeilonYing) [reply commands/getuser.js]: Thanks for the catch.

**Pull request merged**